src/pages/ResearcherReview.js

```diff
@@ -73,7 +79,9 @@ class ResearcherReview extends Component {
     }
 
     this.setState(prev => {
-      prev.formData = researcher;
+      prev.user = user;
```

Comment:
I'm seeing some errors here when clicking on a researcher review link:
```
TypeError: Cannot read property 'name' of undefined
ResearcherReview.render
src/pages/ResearcherReview.js:188
  185 | div({ className: "row margin-top-20" }, [
  186 |   div({ className: "col-lg-12 col-md-12 col-sm-12 col-xs-12" }, [
  187 |     label({ className: "control-label" }, ["Institution Name"]),
> 188 |     div({ id: "lbl_profileInstitution", className: "control-data", name: "profileInstitution", readOnly: true}, [institution.name]),
      | ^  189 |   ])
  190 | ]),
  191 | 
View compiled
▶ 19 stack frames were collapsed.
_callee$
src/pages/ResearcherReview.js:81
  78 |   status = false;
  79 | }
  80 | 
> 81 | this.setState(prev => {
     | ^  82 |   prev.user = user;
  83 |   prev.institution = institution;
  84 |   prev.formData = researcherProps;
View compiled
▶ 5 stack frames were collapsed.
```
<img width="1505" alt="Screen Shot 2021-06-22 at 8 47 30 AM" src="https://user-images.githubusercontent.com/116679/122927092-83ed6500-d336-11eb-9d0e-1f8f9ef79c3f.png">

Reply:
Looks like this is fixed, thank you!